source/_components/remote.harmony.markdown

```diff
@@ -118,28 +131,94 @@ Send a single command or a set of commands to one device, device ID and availabl
 | Service data attribute | Optional | Description |
 | ---------------------- | -------- | ----------- |
 | `entity_id`            |      yes | Only act on a specific remote, else target all.
-| `device`               |       no | Device ID to send the command to.
+| `device`               |       no | Device ID or Device Name to send the command to.
 | `command`              |       no | A single command or a list of commands to send.
 | `num_repeats`          |      yes | The number of times to repeat the command(s).
 | `delay_secs`           |      yes | The number of seconds between sending each command.
 
+In the file 'harmony_REMOTENAME.conf' you can find the available devices and commands, for example:
+
+```text
+{
+    "Devices": {
+        "TV": {
+            "commands": [
+                "PowerOff",
+                "PowerOn"
+            ],
+            "id": "327297814"
+        },
+        "Receiver": {
+            "commands": [
+                "PowerOff",
+                "PowerOn",
+                "VolumeUp",
+                "VolumeDown",
+                "Mute"
+            ],
+            "id": "428297615"
+        }        
+    }
+}
+```
+
+
 A typical service call for sending several button presses looks like this:
 
 ```yaml
 service: remote.send_command
 data:
   entity_id: remote.tv_room
   command:
-    - home
-    - 1
-    - 2
-  device: 4576546
+    - PowerOn
+    - Mute
+  device: Receiver
   delay_secs: 0.6
 ```
+OR
+```yaml
+service: remote.send_command
+data:
+  entity_id: remote.tv_room
+  command:
+    - PowerOn
+    - Mute
+  device: 428297615
+  delay_secs: 0.6
+```
+
+A service call to send a command and hold for 3 seconds would look like:
+```yaml
+service: remote.send_command
+data:
+  entity_id: remote.tv_room
+  command:
+    - Up
+  device: blind
+  hold_secs: 3
+```
+
+### {% linkable_title Service `remote.change_channel` %}
+
+Sends the change channel command to the Harmony HUB 
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            |      yes | Only act on a specific remote, else target all.
```

Comment:
Not a doc point, but I believe Home Assistant is moving away from that, and requiring that `all` be used as the entity ID to target all devices.